src/assets/Comments.jsx

```diff
@@ -11,15 +11,15 @@ export default function Comments() {
     const [lastId, setLastId] = useState(5)
     const [activeComment, setActiveComment] = useState(null)
     const [activeModal, setActiveModal] = useState(null)
-
+    
     React.useEffect(() => {
         localStorage.setItem('comments', JSON.stringify(commentData))
     }, [commentData])
 
     function sortComments() {

```

Comment:
You've implemented this "sortComments" function as reusable but it has a problem. In example: you're using it from "scoreChange" function when you've already called "setCommentData" and changed the state, and then you call "sortComments" and call "setCommentData" from it and change the state yet again. Each state change will trigger a new re-render mechanism in react so the component will re-render twice: once with unsorted comments, and once with sorted. To resolve this you can make just the sort callback function as reusable:
`sortByScore = (a, b) => b.score - a.score`
And then use it in "scoreChange" function like this:
`const newCommentData = prevCommentData.map(com => {...}).sort(sortByScore)`